server/app/routes.py

```diff
@@ -62,16 +62,32 @@ async def modify_watchlist():
 		watchlist.write()
 		return '', 200 # ok
 	
+	download_percentage = 0
```

Comment:
this should be a dictionary with id as key and percentage as value, to handle downloading of 2 different videos at the same time